README.md

```diff
@@ -575,21 +575,26 @@ This repository contains a curated list of awesome open source libraries that wi
 * [OctoML](https://octoml.ai/) - OctoML makes AI more sustainable through efficient model execution and automation to scale services and reduce engineering burden.
+* [OpenAI](https://openai.com/) - OpenAI aims to promote and develop friendly AI in a way that benefits humanity as a whole. 
 * [Pinecone](https://www.pinecone.io/) - Pinecone vector database makes it easy to build high-performance vector search applications
-* [Prodigy](https://prodi.gy/) - Active learning-based data annotation. Allows to train a model and pick most 'uncertain' samples for labeling from an unlabeled pool.
+* [Prodigy](https://prodi.gy/) - Prodigy is a scriptable annotation tool so efficient that data scientists can do the annotation themselves, enabling a new level of rapid iteration.
 * [RapidMiner](https://rapidminer.com/) - RapidMiner is the enterprise-ready data science platform that amplifies the collective impact of your people, expertise and data for breakthrough competitive advantage.
+* [Replicate](https://replicate.com) - Replicate lets you run machine learning models with a cloud API, without having to understand the intricacies of machine learning or manage your own infrastructure.
 * [Robust Intelligence](https://www.robustintelligence.com/) - Robust Intelligence is an end-to-end ML integrity solution that proactively eliminates failure at every stage of the model lifecycle. From pre-deployment vulnerability detection and validation to post-deployment monitoring and protection, Robust Intelligence gives teams the confidence to scale models in production across a variety of use cases and modalities.
 * [Scale](https://scale.com) - Scale AI turns raw data into high-quality training data by combining machine learning powered pre-labeling and active tooling with varying levels and types of human review.
 * [Scribble Enrich](https://www.scribbledata.io/product) - Customizable, auditable, privacy-aware feature store. It is designed to help mid-sized data teams gain trust in the data that they use for training and analysis, and support emerging needs such drift computation and bias assessment.
 * [SigOpt](https://sigopt.com/) - SigOpt is a model development platform that makes it easy to track runs, visualize training, and scale hyperparameter optimization for any type of model built with any library on any infrastructure.
 * [Skymind](https://skymind.global/) - Software distribution designed to help enterprise IT teams manage, deploy, and retrain machine learning models at scale.
 * [Skytree](http://skytree.net) - End to end machine learning platform - [(Video)](https://www.youtube.com/watch?v=XuCwpnU-F1k).
+* [Snowflake](https://www.snowflake.com) - Snowflake is a cloud data warehouse that can store and analyze all your data records in one place. 
```

Comment:
Less of an mlops platform and more of a dataops one

Reply:
Good catch. I've seen that we have added Apache Airflow, Apache Spark, as well as Pachyderm, which are all dataops tools rather than mlops, shall we also remove them @axsaucedo ?

Reply:
No, those are widely used for MLOps orchestration at scale, whereas snowflake is indeed a data warehouse - let's keep those but can remove this one

Reply:
https://www.snowflake.com/trending/machine-learning-tools
https://www.snowflake.com/en/data-cloud/workloads/data-science-ml/
https://www.snowflake.com/guides/machine-learning-platforms

But Snowflake is now already famous for its major contributor as a machine learning tool provider, as you can see from the above news. @axsaucedo

Reply:
These are just examples of how snowflake can be used as part of the data warehouse of data platforms, if you read the last doc it mentions multiple data platforms like data robot, so it's a data warehouse, let's remove

Reply:
but datarobot exist in our list @axsaucedo we might not prefer one to the other since they are serving the same purposes. @axsaucedo

Reply:
Data robot is an mlops platform, snowflake's article talks about "how to use snowflake with ml platforms", an example being data robot - may be best if you read the last article fully as it seems there's some confusion here

Reply:
How about databricks? I cannot tell if databricks differentiate much with Snowflake. But Databricks is in the list. @axsaucedo

Reply:
Databricks only added data warehousing recently with delta lake, but is a vendor for experimentation, production serving of mlflow, and orchestration - either way, highlighting other frameworks won't make snowflake an mlops framework

Reply:
> 

All right, I will remove snowflake @axsaucedo